src/main/java/seedu/address/storage/meeting/JsonAdaptedMeeting.java

```diff
@@ -108,6 +108,14 @@ public Meeting toModelType() throws IllegalValueException {
         }
         final Location modelLocation = new Location(location);
 
-        return new Meeting(modelTitle, modelDescription, modelFrom, modelTo, modelContacts, modelLocation);
+        Meeting meeting;
+
+        try {
+            meeting = new Meeting(modelTitle, modelDescription, modelFrom, modelTo, modelContacts, modelLocation);
+        } catch (IllegalArgumentException e) {
```

Comment:
assuming all caught IllegalArgumentException is related to INCORRECT_FROM_AND_TO_ORDER
could try directly using isValidFromAndTo instead

Reply:
Ahh, I see. Good point

Reply:
Ohh, but on second thought, `isValidFromAndTo() `is executed inside `new Meeting()` already. I think it would be redundant to execute twice?